packages/trees/backend/sirius-components-collaborative-trees/src/main/java/org/eclipse/sirius/components/collaborative/trees/api/TreeConfiguration.java

```diff
@@ -31,7 +31,7 @@ public class TreeConfiguration implements IRepresentationConfiguration {
 
     public TreeConfiguration(String editingContextId, List<String> expanded) {
         String uniqueId = editingContextId + expanded.toString();
-        this.treeId = UUID.nameUUIDFromBytes(uniqueId.getBytes()).toString();
+        this.treeId = "explorer://" + UUID.nameUUIDFromBytes(uniqueId.getBytes()).toString();
```

Comment:
I can understand why this is the shortest way to do what you need but this will not work forever., we will have to explore a better way to specify the identifier of the tree... It can be done later since there are no need to create additional work for this commit.